x-pack/legacy/plugins/infra/public/components/logging/log_text_stream/highlighting.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import euiStyled from '../../../../../../common/eui_styled_components';
+
+export const HighlightMarker = euiStyled.span`
```

Comment:
I brought this up in Slack about the contrast being a little too heavy for highlighted text IMO. Now, I'm sure if we can pass in the SASS mixins here or how we might be able to achieve the [`tintOrShade` mixin](https://elastic.github.io/eui/#/guidelines/sass), but if we can use such a utility to always achieve a good contrast I think the UI will look better for it. I've created a simple Codesandbox to show how it works and @snide made the suggestion to create a `EuiMark` component that can be used in several other cases for highlighting.

<img width="1680" alt="Screenshot 2019-07-01 at 22 26 42" src="https://user-images.githubusercontent.com/4104278/60464599-57267b80-9c4f-11e9-9ebb-b7de4e6b8d64.png">

This example is using the `euiColorAccent` and a 70% tint.
Thoughts?

Reply:
Sorry, I forgot to link to the [Codesandbox](https://codesandbox.io/s/euimark-0f3wx). Just a simple styling of the `<mark>` HTML element, but I think we could easily turn this into a simple component where you could pass a color variable or custom hex code and it will tint/shade accordingly and adapt the overlaying text so it remains legible.

Reply:
Thanks for providing that sandbox :heart: I have incorporated your proposal and updated the screenshots above. We can easily replace the `<HighlightMarker>` with a marker from EUI when it's available.

Reply:
Looks great!